x-pack/README.md

```diff
@@ -13,8 +13,21 @@ Elasticsearch will run with a basic license. To run with a trial license, includ
 Example: `yarn es snapshot --license trial --password changeme`
 
 # Testing
+## Running specific tests
+| Test runner  | Test location                                                                       | Runner command (working directory is kibana/x-pack)                                     |
+| ------------ | ----------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------- |
+| Jest         | `x-pack/**/*.test.js`<br>`x-pack/**/*.test.ts`                                      | `cd x-pack && yarn test:jest -t regexp [test path]`                                     |
```

Comment:
Same feedback as for Kibana - use `scripts/jest` and outline the separate server/runner functional test scripts.